twitterpuller/src/main/java/de/brandwatch/minianalytics/twitterpuller/twitter/TweetSearcher.java

```diff
@@ -0,0 +1,53 @@
+package de.brandwatch.minianalytics.twitterpuller.twitter;
+
+import de.brandwatch.minianalytics.twitterpuller.kafka.Producer;
+import de.brandwatch.minianalytics.twitterpuller.model.Resource;
+import de.brandwatch.minianalytics.twitterpuller.postgres.cache.QueryCache;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Component;
+import twitter4j.*;
+
+import java.sql.Timestamp;
+import java.time.Instant;
+
+@Component
+public class TweetSearcher {
+
+    @Autowired
+    QueryCache queryCache;
+
+    @Autowired
+    Producer producer;
+
+    private static final Twitter twitter = TwitterFactory.getSingleton();
```

Comment:
probably good to make this a bean instead, like you did with `TwitterStream`

Reply:
Do you mean the whole TweetSearcher or only the `Twitter` Object?

Reply:
`TweetSearcher` is a bean already via the `@Component` annotation. I mean `Twitter`.

Reply:
Thats why I was a little bit confused

Reply:
Ok, done